test/e2e/serial/README.md

```diff
@@ -59,7 +59,7 @@ they found it before they run.
 ### Tests tagging 
 
 - Tagging tests to a specific feature functionality were done by adding tags in the spec description like `It("[rtetols][distruptive]...")`.
-Starting Jul 2024, while the master branch is pointing to 4.18, the new preferred way to tag tests is using ginkgo `Label("")` such as `Context("should run pods requesting host-level resources", Label("hostlevel","distruptive"), func(){..})`. Tags aren't forbidden yet; they are just deprecated, so whenever a tag is added, a label is also required.
+Starting May 2025, while the master branch is pointing to 4.20, the new preferred way to tag tests is using ginkgo `Label("")` such as `Context("should run pods requesting host-level resources", Label("hostlevel","distruptive"), func(){..})`. Tags aren't forbidden yet; they are just deprecated, so whenever a tag is added, a label is also required.
```

Comment:
this documents the point in time on which we made a decision, so it should never be updated. History should never be rewritten